script/pip_check

```diff
@@ -3,7 +3,7 @@ PIP_CACHE=$1
 
 # Number of existing dependency conflicts
 # Update if a PR resolves one!
-DEPENDENCY_CONFLICTS=4
+DEPENDENCY_CONFLICTS=5
```

Comment:
Resolve this (some options i mentioned in a comment thread) instead of introducing more conflicts.

Reply:
Attrs should probably be bumped in a separate PR, not this one.

Reply:
Thanks, pulled this out into https://github.com/home-assistant/core/pull/84205